backend/tracim_backend/error.py

```diff
@@ -0,0 +1,53 @@
+# Error code format
+# 1xxx: not found error
+# 2xxx: validation error
+# 3xxx: conflict error
+# 4xxx: authentication and authorization
+# 9xxx: core errors(family of error code reserved
+# for unclassable errors or very low level errors)
+
+
+# Tracim Not found Error
+ERROR_CODE_USER_NOT_FOUND = '1001'
```

Comment:
two last things to change:

- remove the ERROR_CODE prefix which is redondant with the name of the module `error.py`
- use integers instead of strings for err codes.